components/core/header/Header.vue

```diff
@@ -38,22 +46,31 @@ export default {
 
 <style lang="postcss" scoped>
 .header-container {
-    @apply w-full h-full px-4 justify-between items-center;
+    @apply w-full;
     font-size: 20px;
     color: #f0ebf5;
     background-color: #121023;
     height: 64px;
 }
 
-.header-container.--desktop .core-localeSwitch:hover {
+.header-container.--desktop.core-localeSwitch:hover {
     color: #c386ae;
 }
 
-.header-container.--mobile .core-localeSwitch:hover {
+.header-container.--mobile.core-localeSwitch:hover {
     background-color: unset;
 }
 
-.header-container.--mobile .core-localeSwitch {
+.header-container.--mobile.core-localeSwitch {
     z-index: 1002;
 }
+
+.header__nav-bar {
+    @apply w-full flex justify-between items-center px-4;
+    pointer-events: auto; /*enable click-through ScrollableNavBar*/
```

Comment:
Minor suggestion that doesn't block the PR:

```suggestion
     /* enable click-through ScrollableNavBar by `pointer-events: auto` */
    @apply w-full flex justify-between items-center px-4 pointer-events-auto;
```

It looks verbose but using tailwind for consistency is better in my opinion

Reply:
Thanks. I will use this style in the future.